### ISSUE TITLE
Steam can't find libpulsecommon.so

### ISSUE THREAD

**Issue by chilledfrogs** [void-linux/void-packages#27260]
### System
* xuname:  Void 5.9.14_1 x86_64 GenuineIntel uptodate rrrrmdFFFFFFF
* package:  `steam-1.0.0.68_1`

### Expected behavior
Steam launches

### Actual behavior
Fatal error on launch:
```
Error: You are missing the following 32-bit libraries, and Steam may not run:
libpulsecommon-14.0.so
```
Succeeded eventually by:
```
Failed to load steamui.so - dlerror(): libpulsecommon-14.0.so: wrong ELF class: ELFCLASS64
```
So it seems to somehow (only?) see the 64-bit shlib even though I have the 32-bit one as well in `/usr/lib32/pulseaudio/libpulsecommon-14.0.so`.

I can work around this in the meantime by prepending (well really just setting here since I don't have anything in this variable by default) `/usr/lib32/pulseaudio` to `LD_LIBRARY_PATH` when launching `steam` from the command line.

### Steps to reproduce the behavior
Just launch Steam
(Tried `steam --reset` just in case, no dice)

**Comment by Acetsetebos** [comment]
I'm having the same problem except the second error reads:
`Failed to load steamui.so - dlerror(): libpulsecommon-14.0.so: cannot open shared object file: No such file or directory`
I checked and `/usr/lib32/pulseaudio/libpulsecommon-14.0.so` does exist.

**Comment by ericonr** [comment]
The package is being rebuilt and should be fixed soon.

**Comment by sburris0** [comment]
Just tested with `pulseaudio-14.0_2` and it works for me

**Comment by Johnnynator** [comment]
Closed with e496c5682cc9f157f07e99c1007f3c1c8cfa65db
